client/templates/course/lesson/quiz/questions/autoform/afQuestionOptionsGroup.html

```diff
@@ -0,0 +1,26 @@
+<template name="afFieldQuestionOptionsGroup">
+    {{#each displayedOptions }}
+        <div style="{{#if isHidden}}display: none{{/if}}" >
+            <span class="radio-inline checkbox-inline">
+                    {{#if isMultipleAnswer }}
```

Comment:
This is a good approach, and can be generalized to show specific fields based on question type:

```html
{{# if afFieldValueIs name="question.type" value="multipleChoice" }}
  <!-- Show the multiple choice question field -->
  {{> afQuickField name="multipleChoice" }}
{{/ if }}
```

Then, each of the field types would be defined with their own sub-schema. This means we can easily add new question types in the future.